### PR TITLE
Auth start

### PR DESCRIPTION
**Pull request by Mohitkumar33**
wishlist functionality is done using API.
Post, get, delete functionality used
Navbar Login Logout button
The wishlist icon is disappeared if the user is not logged in.
Please give your review

### CODE REVIEW

File: src/pages/Cart.js [hunk 1]
```diff
@@ -64,7 +61,9 @@ const Cart = () => {
                   <button className="card-2-button-1">Go to Cart</button>
                 </div>
                 <div>
-                  <button className="card-2-button-2">Remove from Wishlist</button>
```
Comment (ishapatell1) [inline src/pages/Cart.js:67]: I like how you are refactoring code with all the functionalities. Please take care of extra spaces too.

File: src/pages/Products.js [hunk 2]
```diff
@@ -15,13 +16,18 @@ import {
 } from "../utilitites/index";
```
Comment (ishapatell1) [inline src/pages/Products.js:16]: you may want to fix this typo here. Please check it once

**Review by ishapatell1 — APPROVED** [review]
Your App looks good. The code is clean and everything is well crafted. I will just suggest using better-quality pictures in the app. On your home page pictures look a little distorted but the Product page is splendid. Else, In terms of functionalities, Good work Mohit.

File: src/components/homePage/DealsOfTheDay.js [hunk 3]
```diff
@@ -18,6 +18,7 @@ const DealsOfTheDay = () => {
               <img src={i.image} alt={i.brand} />
               <p>{i.name}</p>
               <p>₹{i.mrp - (i.mrp * i.discount) / 100}</p>
+              <p>{i.discount}% off</p>
```
Comment (ArjunGTX) [inline src/components/homePage/DealsOfTheDay.js:21]: What does this `i` stand for? Couldnt it be a more meaningful variable name?

File: src/pages/Login.js [hunk 4]
```diff
@@ -1,36 +1,89 @@
...
+  const callUserLogin = async (emailId, passw) => {
+    try {
+      const { data } = await axios.post("/api/auth/login", {
+        email: emailId,
+        password: passw,
+      });
+      console.log(data);
+      localStorage.setItem("token", data.encodedToken);
+      localStorage.setItem("userName", data.foundUser.firstName);
+      console.log(localStorage);
```
Comment (ArjunGTX) [inline src/pages/Login.js:23]: This log statement can be removed

**Review by ArjunGTX — APPROVED** [review]
Code looks good in general. Suggested some changes, please have a look

File: src/utilitites/wishlistUtils.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+const setWishList = async () => {
+  try {
+    const config = {
+      headers: {
+        authorization: localStorage.getItem("token"),
+      },
+    };
+    const { data } = await axios.get("/api/user/wishlist", config);
+    console.log("set wishlist function called");
```
Comment (mnath30) [inline src/utilitites/wishlistUtils.js:11]: You can remove the console.log

**Review by mnath30 — APPROVED** [review]
Code looks clean and well structured 
You can remove the console.log wherever not required

File: src/components/homePage/DealsOfTheDay.js (hunk 3 above)
Comment (Mohitkumar33) [reply src/components/homePage/DealsOfTheDay.js:21]: Yes sure

File: src/utilitites/wishlistUtils.js (hunk 5 above)
Comment (Mohitkumar33) [reply src/utilitites/wishlistUtils.js:11]: okk

**Pull request merged**